aebn_dl.py

```diff
@@ -213,7 +211,7 @@ def ffmpeg_error_check(audio_bytes):
             # grab audio segment from the middle of the stream
             data_segment_number = int(self.total_number_of_segments / 2)
             seg_data = self._download_segment("a", data_segment_number, stream_id, return_bytes=True)
-            if not ffmpeg_error_check(seg_0 + seg_data):
+            if not ffmpeg_error_check(seg_0) and not ffmpeg_error_check(seg_data):
```

Comment:
this should be `seg_0 + seg_data`, to make a valid segment
checking them individually won't work

Reply:
Problem is, the function ffmpeg_error_check() is returning a boolean, so Pylance is complaining that `Operator "+" not supported for types "bytes | bool" and "bytes | bool"`. Original line works somehow, but it doesn't seem to be a good syntax.

Reply:
I don't see any problems reported by pylance on my end 🤷 
segment bytes will stay as bytes, because we run download function with a parameter to do it
either way, the check for bad audio won't work with the change you proposed
Maybe remove this change for now, so i'll merge the rest?
i'm fine with changing it, but it shouldn't break the code 😄